framework/sandstone.cpp

```diff
@@ -3311,6 +3311,9 @@ int main(int argc, char **argv)
             } else {
                 sApp->endtime = sApp->starttime + string_to_millisecs(optarg);
             }
+            sApp->shmem->use_strict_runtime = true;
```

Comment:
This is a behaviour change. Why is this here?

Reply:
If strict runtime is not enforced, it's not stopping on a truly infinite supply of the tests, even if allocated time (with `-T`) has expired.

Reply:
We'll need to document this.